LocalMultiplayerAgent/Program.cs

```diff
@@ -37,9 +36,14 @@ public static async Task Main(string[] args)
             // lcow stands for Linux Containers On Windows => https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/linux-containers
             Globals.GameServerEnvironment = args.Contains("-lcow") && RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
                 ? GameServerEnvironment.Linux : GameServerEnvironment.Windows; // LocalMultiplayerAgent is running only on Windows for the time being
+            Globals.CreateDeployment = args.Contains("-deploy");
+
             MultiplayerSettings settings = JsonConvert.DeserializeObject<MultiplayerSettings>(File.ReadAllText("MultiplayerSettings.json"));
 
-            settings.SetDefaultsIfNotSpecified();
+            if (!Globals.CreateDeployment)
```

Comment:
why is the variable global? can we have it as a local variable? Global variables are hard to test and error prone